docs/remote/containers.md

```diff
@@ -568,11 +646,7 @@ VS Code will then **automatically use both files** when starting up any containe
 docker-compose -f docker-compose.yml -f .devcontainer/docker-compose.extend.yml up
 ```
 
-### Using an updated Dockerfile to automatically install more tools
-
-You may want to install other tools such as Git inside the container for the service you've specified. You can easily [do this manually](#installing-additional-software-in-the-sandbox). However, you can also create a custom `Dockerfile` specifically for development that includes these dependencies. The [vscode-dev-containers repository](https://github.com/Microsoft/vscode-dev-containers) contains examples you can use to augment a copy of a Dockerfile or when creating a new one.
-
-Assuming you put this file under `.devcontainer/Dockerfile`, the `.devcontainer/docker-compose.yml` above would be modified as follows:
+While the `postCreateCommand` property allows you to install additional tools inside your container, in some cases you may want to have a specific Dockerfile for development. You can also use this same approach to reference a custom `Dockerfile` specifically for development without modifying your existing Docker Compose file.  For example, you can update `.devcontainer/devcontainer.extend.json` as follows:
```

Comment:
Should the filename be `docker-compose.extend.yml`?

Reply:
Good catch - that's actually been there a while and I totally missed it on re-reads.